docker-compose.yml

```diff
@@ -53,14 +48,7 @@ services:
       PMA_USER: root
       PMA_PASSWORD: $DB_PASSWORD
     networks:
-      elonet:
-        ipv4_address: 172.16.11.5
+      - elonet
 networks:
   elonet:
-    driver: bridge
-    ipam:
-      driver: default
-      config:
-      - subnet: 172.16.11.0/28
```

Comment:
Te wszystkie IP są określone nie bez powodu, aby nie wchodziły w konflikt z jakąkolwiek siecią lokalną programisty. Czy usunięcie tych adresów jest wymagane przez v3 docker-compose?

Reply:
Ale to nie jest wewnętrzna sieć dockera?

Reply:
No to właśnie ma być wewnętrzna sieć Dockera i ustalona dla niej przestrzeń adresów.